src/Makefile

```diff
@@ -0,0 +1,22 @@
+# $FreeBSD$
+
+MAN=
+CFLAGS=	-Wall -lmd -lz -lfetch -I..
+
+LDFLAGS+=	${.CURDIR}/../lib/libogit.a
```

Comment:
On the other hand, you're not actually using any shared library you intend to build (I assume you plan to change that at some point?).

This LDFLAGS isn't right for OpenBSD if you do the usual `make obj && make` invocation. I assume FreeBSD also uses some sort of objdir for building? Not a huge deal right now but something to keep in mind if this ever gets imported into any trees.

Reply:
Right, all of my testing is just dumping objs into .CURDIR for now- I'll fix this stuff in another iteration on this PR.

Reply:
Changed this to `${.OBJDIR}../lib/libogit.a` for now. Future iterations of this will switch to the shared lib, and tests will be added around here as well.

Speaking of tests, figuring out a nice OS-independent test framework is on my list somewhere...